src/muya/lib/utils/exportHtml.js

```diff
@@ -161,7 +161,7 @@ class ExportHtml {
    * @param {*} title Page title
    * @param {*} printOptimization Optimize HTML and CSS for printing
    */
-  async generate (title = '', printOptimization = false) {
+  async generate (title = '', printOptimization = false, extraCss = '') {
```

Comment:
For what is `extraCss` used for?

Reply:
We can select theme or adjust the padding and page size when we export HTML, so I add this `extraCss`.